src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -448,30 +417,68 @@ public void sendEmailWhenOrderIsServed(String authorisation,
             //PRL-4225 - set bulkIds in the orderCollection & update in caseDataMap
             addBulkPrintIdsInOrderCollection(caseData, bulkPrintOrderDetails);
             caseDataMap.put(ORDER_COLLECTION, caseData.getOrderCollection());
+            sendEmailToCafcassCymru(caseData,listOfOtherAndCafcassEmails,authorisation, orderDocuments);
```

Comment:
are we sending email to anyother party than cafcass? if not why the name is listOfOtherAndCafcassEmails?